tests/test_cerebras_response_validation.py

```
@@ -0,0 +1,213 @@
+
+
+def test_cerebras_api_reachable():
+    """Test that we can reach Cerebras API with credentials"""
+    api_key = os.getenv("CEREBRAS_API_KEY")
+    base_url = os.getenv("CEREBRAS_BASE_URL", "https://api.cerebras.ai/v1")
+
+    if not api_key:
+        pytest.skip("CEREBRAS_API_KEY not set - cannot test Cerebras integration")
+
+    import requests
+    response = requests.get(
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)